_posts/theme/2020-8-18-Nord.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: theme
+title: Nord
+category: theme
+homepage: https://github.com/ChristosBouronikos/typora-nord-theme/
+download: https://github.com/ChristosBouronikos/typora-nord-theme/releases/download/1.0/typora-nord-theme.zip
+built-in: false
+author: Christos Bouronikos
+thumbnail: nord.jpg
```

Comment:
you are using nord.png, but here you wrote jpg

Reply:
dammit, thanks! will fix asap!